### ISSUE TITLE
Edit ----associate photo editing site

### ISSUE THREAD

**Issue by bai12123** [d2phap/ImageGlass#1347]
### Is your feature request related to a problem? Please describe.
I am not an in-depth picture worker, for picture editing, I have not installed any picture editing software, just an online picture editing website can solve my needs, like  www.fotor.com.cn/photo-editor  ；photopea.com。I want imageglass to be associated with image editing site

### Describe the solution you'd like
Usually I install these image editing sites as chrome apps. photopea.com also can be installed as a chrome app, I want to associate this. When I click "edit image", the photopea app opens automatically, that's all.Then I can import the image to edit

### Describe alternatives you've considered
If it is not possible to associate a chrome app, it is also possible to associate an online image editing website

### Additional context
![20-35-57 ](https://user-images.githubusercontent.com/99849852/170700630-300ce998-7c89-4ffe-b2a1-c8221101dbfb.png)
![20-36-28 ](https://user-images.githubusercontent.com/99849852/170700636-970d691f-ea6e-47ae-81f8-c476e7d39fa0.png)
@d2phap  thanks

**Comment by bai12123** [comment]
When will the official version 9.0 be released?It would be great if this feature was implemented in 9.0

**Comment by d2phap** [comment]
### Short answer:
- This feature must support from the Chrome app, not ImageGlass.
- Chrome app should support reading arguments command line.

### Long answer:

You can launch Chrome app in ImageGlass by copying the Target command of the app.
Let's say I installed Youtube Chrome app, and add it to my desktop. Right-click to open context menu, select Properties, copy the `Target` field:

![image](https://user-images.githubusercontent.com/3154213/170707197-e940ac3d-1613-4188-8678-4db5f9166f5f.png)

The Target field value is something like this.
```
"C:\Program Files (x86)\Microsoft\Edge\Application\msedge_proxy.exe" 
--profile-directory="Profile 2" 
--app-id=agimnkijcaahngcdmfeangaknmldooml 
--app-url=https://www.youtube.com/?feature=ytca 
--app-launch-source=4
```

There are basically 2 parts:
- Executable: which is `C:\Program Files (x86)\Microsoft\Edge\Application\msedge_proxy.exe`,
- and the arguments are `--profile-directory="Profile 2" --app-id=agimnkijcaahngcdmfeangaknmldooml --app-url=https://www.youtube.com/?feature=ytca --app-launch-source=4`

Now open ImageGlass Settings > Edit, choose the extension to edit. Copy executable path to `App path` box, and parameters to the `App arguments` box. ** Put the `"<file>"` variable in a correct position of the argument**. Click OK, and Save button. Now when you click on Edit menu, it should launch the Chrome app.

![image](https://user-images.githubusercontent.com/3154213/170709061-30efc1ae-f376-41c8-9455-09107c70ef55.png)

------------
### Important note!!
### If the Chrome app **SUPPORTS** reading from arguments, it will open it.
In my tests, there is not any Chrome app support this. So my current answer is "no, it's not possible". The Chrome app must work on a solution to support arguments.

**Comment by d2phap** [comment]
> When will the official version 9.0 be released?It would be great if this feature was implemented in 9.0

Version 9.0 will have 2 or 3 more beta, before it can be official released.

**Comment by bai12123** [comment]
Thanks for your help.I just need launch Chrome app in ImageGlass。and I successfully opened the image in the chrome app
